cmd/kfulint/main.go

```diff
@@ -107,13 +108,13 @@ func (l *linter) LoadProgram() {
 	l.prog = prog
 }
 
-func (l *linter) InitContext() {
+func (l *linter) initContext() {
```

Comment:
Usually, capitalized (exportes) methods on unexported types are OK to be without a comment, and go lint does not trigger on them. The reason it's this way is to emphasize that those methods are expected to be called from outside of the type, as opposed to unexported methods that should be called only inside type itself (like private methods).
I think we can even write a checker for this: if unexported type method is ever called outside of it's own methods, suggest making it "exported", otherwise suggest making it "unexported" (if it's only called from within that type).
This check can be disabled by people who don't like this convention, but it may help for projects that try to comply with it.

The main benefit is: when you're making that type exported, your API is already almost done, what is left is to add comments to exported methods. If all methods are unexported, you will need to do a manual job.